### PR TITLE
add view

### PR DESCRIPTION
**Pull request by ariririri**
I change migration, which connect foreign key and index view to show all questions

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,7 +2,8 @@
 # pipenv environment
 Pipfile*
 .envrc
-.vocode
+.vscode
```
Comment (chaspy) [inline .gitignore]: Could you use `.git/info/exclude` instead `.gitignore` to ignore your private file?
https://medium.com/@dave_lunny/exclude-files-from-git-without-committing-changes-to-gitignore-986fa712e78d
Or Try `$HOME/.config/git/ignore` for your global setting.

File: .gitignore [hunk 2]
```diff
@@ -2,7 +2,8 @@
...
+todo.md
```
Comment (chaspy) [inline .gitignore]: Same as https://github.com/chaspy/ask-python/pull/22/files#r272825050

**Comment by chaspy** [comment]
## What I did
- `./run.sh`
- migration: `docker exec ask-python_web_1 python manage.py migrate`
  - It should be run automated ref: #23 
- Create user at `/users/signup`
- Login
- Create a question
- Answer the question

Looks good to me 👍

**Comment by ariririri** [comment]
@chaspy  sorry to be late.
I write my local files `.git/info/exclude`
and I add the function to display username when you answer.
Please review and merge.

**Comment by chaspy** [comment]
Confirmed showing username when answered a question 👍 
Screenshot 2019-04-13 14 04 26

**Comment by chaspy** [comment]
@ariririri 
https://github.com/chaspy/ask-python/pull/22/commits/eafd9b9fadec59d56b6ac35b98966b93eea96504

Removing git exclude and showing user names should be separate commits.
Please consider in future.

**Pull request merged**